app/controllers/admin/playlist_activities_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Admin::PlaylistActivitiesController < Admin::ResourceController
+  def new
+    @student = Student.find(params[:student_id])
+    @resource = @student.playlist.playlist_activities.new
```

Comment:
I think you get at this in the comments already, but, I'm not sure we need the playlist abstraction here. I think a foreign_key on playlist_activities gets us the same thing, and makes the query less indirect, no? Also, in this case, would this not be the same as `PlaylistActivity.new(playlist: student.playlist)`